Integrations/CentralBankIntegrationLib/CentralBankNewsPublisher.cs

```diff
@@ -20,8 +21,19 @@ public string GetNews()
             List<Valute> valutes = GetTopValutes();
 
             var sb = new StringBuilder();
-            sb.AppendLine("Top - 5 exchange rates:");
-            sb.AppendLine(valutes != null ? string.Join('\n', valutes) : Errors.CanNotGetCurrencyRateFromCB);
+            // sb.AppendLine(valutes != null ? string.Join('\n', valutes) : Errors.CanNotGetCurrencyRateFromCB);
+            if (valutes.Count == 0)
+            {
+                sb.AppendLine("Valutes are not founded");
+            }
+            else
+            {
+                sb.AppendLine("Top - 5 exchange rates:");
+                foreach (Valute valute in valutes)
+                {
+                    sb.AppendLine(valute.Description());
+                }

```

Comment:
Why not use var?After all, the type is specified explicitly

Reply:
like this? 
var description = valute.Description();
sb.appendline(description? 

It isn't a problem, I can change, but why?

Reply:
You must adhere to your chosen code style.